package.json

```
@@ -80,7 +80,7 @@
     "vhost": "~3.0.2"
   },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": "^8.9.0 || ^10.13.0 || >=12.0.0"
```

```suggestion
    "node": "^8 || ^10 || ^12 || ^14 || ^16 || ^18 || ^20 || >=22"
```

See: https://github.com/expressjs/express/pull/3948#issuecomment-2449356236